lambda/models/state_machine/create_model.py

```diff
@@ -97,14 +99,61 @@ def handle_poll_docker_image_available(event: Dict[str, Any], context: Any) -> D
 def handle_start_create_stack(event: Dict[str, Any], context: Any) -> Dict[str, Any]:
     """Start model infrastructure creation."""
     output_dict = deepcopy(event)
+    request = CreateModelRequest.validate(event)
+
+    prepared_event = {}
+
+    def camelize_object(o):
+        o2 = {}
+        for k in o:
+            fixed_k = k[:1].lower() + k[1:]
+            if ( isinstance(o[k], dict) ):
+                o2[fixed_k] = camelize_object(o[k])
+            else:
+                o2[fixed_k] = o[k]
+        return o2
+            
+    prepared_event = camelize_object(event)
+    prepared_event["containerConfig"]["image"] = {
+        "repositoryArn": os.environ["ECR_REPOSITORY_ARN"],
+        "tag": event["image_info"]["image_tag"],
+        "type": "ecr"
+    }
+
+    response = lambdaClient.invoke(
+        FunctionName=os.environ['ECS_MODEL_DEPLOYER_FN_ARN'],
+        Payload=json.dumps({"modelConfig": prepared_event}),
+    )
+
+    payload = response["Payload"].read()
+    payload = json.loads(payload)
+    stack_name = payload.get('stackName')
+    output_dict["stack_name"] = stack_name
+    model_table.update_item(
+        Key={"model_id": request.ModelId}, AttributeUpdates={"StackName": {"Value": stack_name, "Action": "PUT"}}
```

Comment:
AttributeUpdates is deprecated. Can we migrate to UpdateExpression instead?

https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/LegacyConditionalParameters.AttributeUpdates.html

https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/LegacyConditionalParameters.AttributeUpdates.html#UpdateExpression.instead


I think I may have an example of what that would look like in the DeleteModel workflow

Reply:
will do